src/utils/getLocationsApiData.js

```
@@ -0,0 +1,39 @@
+const { createError } = require('~/utils/errorsHelper')
+const { INTERNAL_SERVER_ERROR } = require('~/consts/errors')
+const API_URL = 'https://countriesnow.space/api/v0.1'
+
+const getCountriesData = async () => {
+  try {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)